Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -34,9 +32,25 @@ public struct Configuration: Equatable {
     public let rules: [Rule]
     public var rootPath: String?              // the root path to search for nested configurations
     public var configurationPath: String?     // if successfully loaded from a path
-    public var hash: Int?
     public let cachePath: String?
 
+    public var cacheDescription: String {
+        let cacheRulesDescriptions: [String: Any] = rules.reduce([:]) { accu, element in
+            var accu = accu
+            accu[type(of: element).description.identifier] = element.configurationDescription
+            return accu
+        }
+        let dict: [String: Any] = [
+            "root": rootPath ?? FileManager.default.currentDirectoryPath,
+            "rules": cacheRulesDescriptions
+        ]
+        if let prettyJSONData = try? JSONSerialization.data(withJSONObject: dict),
```

Comment:
doesn't need to be pretty actually, since I'm just using this for serialization. fixed variable name.